HeadsAnywhere plugin compatibility (#143)
LoneDev6/Wiki-ItemsAdder: compatibility-with-other-plugins/compatible/headsanywhere.md

# HeadsAnywhere Compatibility

> How to show head of players in chat, signs, books, and more using HeadsAnywhere plugin.

Reviewer: ```suggestion
How to show the head of a player in chat, signs, books, and more using the HeadsAnywhere plugin.
```

## [Download Here](https://www.spigotmc.org/resources/headsanywhere-use-players-heads-anywhere.109951/)

> HeadsAnywhere is a plugin that allows you to use player heads as character in your chat, making it easy to access and display player heads anywhere in the game.
> 
> It is compatible with ItemsAdder, allowing you to use custom player heads as well.
> 
> You will need to disable their resourcepack in their config.yml and merge its resourcepack with ItemsAdder's resourcepack to use custom player heads.
> 
> Editation of their default.json to get more accurate head offsets

Reviewer: ```suggestion
HeadsAnywhere is a plugin that allows you to display player heads in your chat, making it easy to access and display player heads anywhere in the game.

It is compatible with ItemsAdder, allowing you to use custom player heads as well.

You will need to disable their resourcepack in their config.yml and merge their resourcepack with ItemsAdder's resourcepack to use custom player heads.

You can edit their default.json file to get more accurate player head offsets.
```
